XCUITests/HomeButtonTests.swift

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class HomeButtonTests: BaseTestCase {
```

Comment:
When a new test is create it is enabled by default in all schemes. Could you please disable it from the SmokeTest or the FullFuncional test plan according to where it is run manually? 

Thanks!

Reply:
Oh this is good to know for new tests. I disabled everywhere except `Fennec_Enterprise_XCUITests`. Let me know if the is correct.

Reply:
Yes, that should be fine, thanks!